bedrock/firefox/templates/firefox/new/quantum/scene1.html

```diff
@@ -27,8 +27,13 @@
 
 {% block main_content %}
 <header>
+{% if LANG.startswith('en') %}
```

Comment:
Doesn't it make more sense to put it behind a tag, if the goal is to expose it later? We can just wait to expose the string on our side, and there will be no code changes needed.

Only downside: en-GB and en-ZA would show the old tagline. Truth is, it's highly unlikely they'll translate the page, so they'll fall back to en-US anyway.

Reply:
Ok will use a tag 👍